misk-web/settings.gradle

```diff
@@ -0,0 +1,21 @@
+//pluginManagement {
```

Comment:
what's up with all this commented out stuff? Remove it or leave a note why it's commented out?

Reply:
Oops I thought I had gotten all of it, I'll remove it and force push.